src/Model/AutoSegmentation.py

```diff
@@ -0,0 +1,180 @@
+import os
+import shutil
+import tempfile
+import logging
+from src.Model.PatientDictContainer import PatientDictContainer
+from src.Model.Worker import SegmentationWorkerSignals
+import fnmatch
+from totalsegmentator.python_api import totalsegmentator
+from src.Model.NiftiToRtstructConverter import nifti_to_rtstruct_conversion
+from src.View.util.RedirectStdOut import ConsoleOutputStream, redirect_output_to_gui, setup_logging
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
We might want to look at ensuring this is set at the very beginning of the OnkoDICOM application (the logging configuration).  I find it useful to have the line number available in logging as well.  For when you need to know "what was the last thing that *did* work".